crypto/src/signature.rs

```diff
@@ -67,7 +67,7 @@ impl Signature {
     /// # Errors
     /// Fails if signing fails
     #[cfg(feature = "std")]
-    fn new(key_pair: KeyPair, payload: &[u8]) -> Result<Self, Error> {
+    pub fn new(key_pair: KeyPair, payload: &[u8]) -> Result<Self, Error> {
```

Comment:
I still think that we can do things without this change

Reply:
Hiding a constructor method usually means that the constructor is unnecessary, and that the type should have been private.